go.mod

```diff
@@ -7,23 +7,23 @@ require (
 	github.com/ProtonMail/go-crypto v0.0.0-20211221144345-a4f6767435ab
 	github.com/cyphar/filepath-securejoin v0.2.2
 	github.com/fluxcd/go-git-providers v0.5.4
-	github.com/fluxcd/helm-controller/api v0.17.2
-	github.com/fluxcd/image-automation-controller/api v0.20.1
-	github.com/fluxcd/image-reflector-controller/api v0.16.0
-	github.com/fluxcd/kustomize-controller/api v0.21.1
-	github.com/fluxcd/notification-controller/api v0.22.3
-	github.com/fluxcd/pkg/apis/meta v0.10.2
+	github.com/fluxcd/helm-controller/api v0.18.0
+	github.com/fluxcd/image-automation-controller/api v0.21.0
+	github.com/fluxcd/image-reflector-controller/api v0.17.0
+	github.com/fluxcd/kustomize-controller/api v0.22.0
+	github.com/fluxcd/notification-controller/api v0.23.0
+	github.com/fluxcd/pkg/apis/meta v0.12.1
 	github.com/fluxcd/pkg/kustomize v0.0.2
```

Comment:
```suggestion
	github.com/fluxcd/pkg/kustomize v0.3.2
```

Reply:
```
$ go get -d github.com/fluxcd/pkg/kustomize@v0.3.2
go get: github.com/fluxcd/pkg/kustomize@v0.3.2: invalid version: unknown revision kustomize/v0.3.2
```

Updating to `github.com/fluxcd/pkg/kustomize@v0.0.3` instead.